package.json

```diff
@@ -18,5 +22,10 @@
     "basic-auth": "^2.0.1",
     "passport": "^0.4.1",
     "unleash-server": "^3.10.1"
+  },
+  "devDependencies": {
+    "execa": "^5.0.0",
+    "tempy": "^1.0.0",
+    "yargs": "^16.2.0"
```

Comment:
this introduces some dev dependencies. this functionality is obviously doable without them, but it would require a fair bit more code. is introducing dev deps a problem?

Reply:
I'm ok with dev-deps here.